graphql_compiler/tests/test_compiler.py

```diff
@@ -5834,27 +5834,7 @@ def test_fold_and_traverse(self):
             ])}
         """
         # TODO: implement multiple traversals in a separate PR
-        expected_sql = NotImplementedError
-        # expected_sql = '''
-        #     SELECT
-        #         [Animal_1].name as animal_name,
-        #         coalesce(folded_subquery_1.fold_output_1, ARRAY[]::VARCHAR[])
-        #             AS sibling_and_self_names_list
-        #     FROM
-        #         db_1.schema_1.[Animal] AS [Animal_1]
-        #     LEFT JOIN (
-        #         SELECT
-        #             [Animal_2].uuid,
-        #             array_agg([Animal_4].name) as sibling_and_self_names_list
-        #         FROM db_1.schema_1.[Animal] AS [Animal_2]
-        #         JOIN db_1.schema_1.[Animal] AS [Animal_3]
-        #         ON [Animal_2].parent = [Animal_3].uuid
-        #         JOIN db_1.schema_1.[Animal] AS [Animal_4]
-        #         ON [Animal_3].uuid = [Animal_4].parent
-        #         GROUP BY [Animal_2].uuid
-        #     ) AS folded_subquery_1
-        #     ON [Animal_1].uuid = folded_subquery_1.uuid
-        # '''
+        expected_mssql = NotImplementedError
```

Comment:
This one isn't supported at the moment?

Reply:
This commented out `expected_sql` that got deleted is largely similar to the `expected_postgresql` below. Traversals inside of folds are not yet implemented for MSSQL, but the `TODO` needs to be updated to reflected that.